geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommand.java

```diff
@@ -289,8 +302,30 @@ private void checkForAsyncQueue(String regionName, boolean synchronous, CacheCon
     }
   }
 
+  private void checkForAEQIdForAccessor(String regionName, boolean synchronous,
+      RegionConfig regionConfig)
+      throws PreconditionException {
+    if (!synchronous) {
+      RegionAttributesType regionAttributesType = regionConfig.getRegionAttributes();
+      boolean isAccessor = MappingCommandUtils.isAccessor(regionAttributesType);
+      if (!isAccessor) {
+        return;
+      } else {
+        String queueName = MappingCommandUtils.createAsyncEventQueueName(regionName);
+        if (regionAttributesType.getAsyncEventQueueIds() != null && regionAttributesType
```

Comment:
Better add null check here, `regionAttributesType` and `getAsyncEventQueueIds ()` can be null.